application-task-ui/src/main/resources/TaskManager/TaskManagerTranslations.xml

```diff
@@ -117,6 +117,20 @@ taskmanager.livetable.assignee=Assignee
 taskmanager.livetable.duedate=Due Date
 taskmanager.livetable.creationDate=Creation Date
 
+# Notifications for changed tasks
+taskmanager.events.taskChangedEvent.description=A task which you are assigned to or you are watching was modified
```

Comment:
This description is not how the notifications currently work. I couldn't get Targetable events to work with notifications for watched pages (would duplicate notifs and require 2 event classes for the same conceptual event). Now the only way to receive notifications for your tasks is to watch the entire task space/wiki, but then you also get notifications for all other tasks.

Would it be ok to add a listener which auto-watches task pages when a new assignee is added? This way, there's only one way to receive notifications, and they are more configurable (assignees can opt out of watching one of their tasks, outside people can watch pages they are not assigned to)

Reply:
I'm not sure I understand exactly the use case you are referring to
Right now, to receive notifications for the task events, you need to subscribe to the event from you profile. This will send you notifications for all changes, no matter if you are assigned or not.
You are looking now for a way to filter these, right? So I will get notifications only for the tasks were I am assigned (probably should work for `reporter` as well), but also if I am watching a specific task (were I am not assigned in some way). Is this the case?

Reply:
The subscribed event only allows you to receive events on pages you are already watching. So if you don't watch the TaskManager space (or the space of all tasks), then you won't receive any notifications.

This is the issue I was facing: If a user doesn't want to watch the TaskManager space (and get 'spam' notifications for tasks of other users), then there's no way they would know when a new task is created/assigned (unless they specifically go to the new task page and manually watch it).